src/bundle/Resources/public/scss/_buttons.scss

```diff
@@ -69,3 +69,45 @@
     align-items: center;
     justify-content: center;
 }
+
+.ez-modal {
+    &--send-to-trash {
+        .modal-header {
+            .close {
+                position: absolute;
+                right: 1rem;
+            }
+        }
+
+        .modal-footer {
+            .btn {
+                line-height: 1.25;
+                padding: .5rem .75rem;
+            }
+
+            .btn-secondary {
+                background-color: $ez-color-base-medium;
+                border-color: $ez-color-base-medium;
+
+                &:hover {
+                    background-color: darken($ez-color-base-medium, 15%);
+                    border-color: darken($ez-color-base-medium, 15%);
+                }
+            }
+        }
+    }
+}
+
+.ez-content-view,
+.ez-trash-list-view {
+    .ez-modal {
+        &--send-to-trash {
```

Comment:
I think one level of nesting might be omitted by setting `.ez-modal--send-to-trash`

Reply:
It can be improved here and there.